src/plugin/tasks/serverless/serverless-s3-api.json

```diff
@@ -1,5 +1,5 @@
 {
-  "target": "serverless/s3/api/${bucket}.yml",
+  "target": "serverless/api/s3/${bucket}.yml",
```

Comment:
Should this file be renamed to account for this change? `serverless-api-s3.json`?

Reply:
It's not consistent... but it makes it easier to see the grouping

Reply:
But maybe we should jsut stick with that... let me see